doc/installation.rst

```diff
@@ -255,39 +237,76 @@ Same as above but use yum rather than apt-get when necessary.
+Anaconda_ is probably the easiest way to install the dependencies that you need.
 
-To install for the system::
+Start a command shell like ``cmd`` or Powershell_ and change directory into the
+*dipy source root directory*.
 
-    python setup.py install
+To install into your system::
 
-To build dipy in the source tree (locally) so you can run the code in the source tree (recommended for following the latest source) run::
+    python setup.py install --compiler=mingw32
 
-    python setup.py build_ext --inplace
+To install inplace - so that dipy is running out of the source code directory::
 
-add the *dipy source root directory* into your ``PYTHONPATH`` environment variable. Search google for ``PYTHONPATH`` for details or see `python module path`_ for an introduction.
+    python setup.py develop
 
-When adding dipy_ to the ``PYTHONPATH``, we usually add the ``PYTHONPATH`` at
-the end of ``~/.bashrc`` or (OSX) ``~/.bash_profile`` so we don't need to
-retype it every time. This should look something like::
+(this is the mode we recommend for following the latest source code).  If you
+get an error with ``python setup.py develop`` make sure you have installed
+`setuptools`_.
 
-  export PYTHONPATH=/home/user_dir/Devel/dipy:/home/user_dir/Devel/nibabel
+If you get an error saying  "unable to find vcvarsall.bat" then you need to
+create a file called "pydistutils.cfg" in notepad and give it the contents ::
 
-After changing the ``~/.bashrc`` or (OSX) ``~/.bash_profile`` try::
+  [build]
+  compiler=mingw32
 
-  source ~/.bashrc
+Save this into your system python ``distutils`` directory as ``distutils.cfg``.
+This will be something like ``C:\Python26\Lib\distutils\distutils.cfg``.
 
-or::
 
-  source ~/.bash_profile
+OSX
+---
+
+Make sure you have Xcode_ and Anaconda_ installed.
+
+From here follow the :ref:`install-source-nix` instructions.
+
+OpenMP with OSX
+---------------
+OpenMP_ is a standard library for efficient multithreaded applications. This 
+is used in Dipy for speeding up many different parts of the library (e.g., denoising 
+and bundle registration). If you do not have an OpenMP-enabled compiler, you can 
+still compile Dipy from source using the above instructions, but it might not take 
+advantage of the multithreaded parts of the code. To be able to compile 
+Dipy from source with OpenMP on Mac OSX, you will have to do a few more things. First 
+of all, you will need to install the Homebrew_ package manager. 
+
+Next, if you are using Anaconda, you will need to run the following::
+    
+	brew reinstall gcc --without-multilib
+	
+This should take about 45 minutes to complete. Then add to your bash 
+configuration (usually in ``~/.bash_profile``), the following::
+
+	export PATH="/usr/local/Cellar/gcc/5.2.0/bin/gcc-5:$PATH
+
+If you are already using the Homebrew Python, or the standard python.org Python, 
+you will need to run::
+	
+	brew install clang-omp
+
+And then edit the ``setup.py`` file to include the following line::
```

Comment:
Where to include it? Maybe it will be nice to say at which line or at the top of the file or something.